frame/contracts/src/lib.rs

```diff
@@ -467,9 +472,11 @@ pub mod pallet {
 			origin: OriginFor<T>,
 			code: Vec<u8>,
 			storage_deposit_limit: Option<<BalanceOf<T> as codec::HasCompact>::Type>,
+			determinism: Determinism,
```

Comment:
I decided not worth the hassle because no UI really uses this dispatchable. Neither polkadot.js nor contracts UI. They both use `instantiate_with_code` exclusively.